quantum/nodes/measure/measure.html

```diff
@@ -125,15 +137,20 @@
     <input type="text" id="node-input-name" placeholder="Name" />
   </div>
   <div class="form-row" hidden>
-    <label for="node-input-classicalRegistersOrBits"><i class="fa fa-check"></i>Are classical registers/bits set?</label>
-    <input type="text" id="node-input-classicalRegistersOrBits" disabled/>
-  </div>
-  <div id="reg-form">
+    <label for="node-input-classicalRegistersOrBits"
+      ><i class="fa fa-check"></i>Are classical registers/bits set?</label
+    >
+    <input type="text" id="node-input-classicalRegistersOrBits" disabled />
   </div>
+  <div id="reg-form"></div>
 </script>
 
 <script type="text/html" data-help-name="measure">
   <p>
-    Quantum measure node
+    This node measures the value of a qubit in order to convert the
+    superpositioned qubit to a classical bit of value 0 or 1. <br /><br />
+    To use this node you must have a classical register enabled in the circuit
```

Comment:
You don't need a classical register to use it.

You simply need to have initialised your circuits with classical bits, using registers or not.